.travis.yml

```diff
@@ -1,50 +1,36 @@
-#
-# Travis configuration guide for R language:
-#   https://docs.travis-ci.com/user/languages/r/
-#   https://github.com/craigcitro/r-travis/wiki/Porting-to-native-R-support-in-Travis
-#
 language: r
 dist: trusty
 sudo: required
 cache: packages
 warnings_are_errors: true
-r_check_args: ""
-# empty r_check_args to turn off --as-cran (on by default) as that can be slow
```

Comment:
`_R_CHECK_CRAN_INCOMING_REMOTE_=false` defined below should resolve problem of slow `--as-cran`